library/src/commonMain/kotlin/com/myunidays/segmenkt/AppsflyerIntegration.kt

```diff
@@ -0,0 +1,10 @@
+package com.myunidays.segmenkt
+
+import com.myunidays.segmenkt.integrations.Integration
+import com.myunidays.segmenkt.integrations.IntegrationFactory
+
+expect class AppsflyerIntegration : Integration<AppsflyerIntegration> {
+    companion object {
+        fun factory(delegate: Any? = null, deeplinkHandler: Any? = null): IntegrationFactory
```

Comment:
this is important, so we now allow for delegate and deeplink handler, which is an iOS only thing